packages/runtime-dom/src/directives/vModel.ts

```diff
@@ -85,6 +85,7 @@ export const vModelText: ModelDirective<
         return
       }
     }
+    if (value === undefined || value === null) value = ''
```

Comment:
Literally assigning NULL to native input value should has same effect. Probably checks `value === null` and `value !== null` is not required.

Check out example:
1. Open: [https://codesandbox.io/s/native-input-value-nwjoz?file=/index.js](https://codesandbox.io/s/native-input-value-nwjoz?file=/index.js)
2. See console logs